src/types.ts

```diff
@@ -104,11 +104,6 @@ export type KeyPair = {
   publicKey: PublicKey
   secretKey: SecretKey
 }
-export type RoleDetails = {
-  name: string
-  capabilities: string[]
-}
-export type AvailableRoles = RoleDetails[]
```

Comment:
These seemed completely unused (even [across the org][0]). I removed them rather than renaming them.

[0]: https://github.com/search?q=org%3Adigidem+availableroles&type=code

Reply:
Yes I think this is from older code, fine to remove.